fbw-a380x/src/systems/instruments/src/MsfsAvionicsCommon/providers/MfdSurvPublisher.ts

```diff
@@ -6,8 +6,12 @@
  * Transmitted from MFD to RMP
  */
 export interface MfdSurvEvents {
-    /** (MFD SURV -> RMP) Is AUTO (true) or STBY (false). */
-    mfd_xpdr_set_auto: boolean,
-    /** (MFD SURV -> RMP) Altitude reporting ON/OFF */
-    mfd_xpdr_set_alt_reporting: boolean,
+  /** (MFD SURV -> RMP) Is AUTO (true) or STBY (false). */
+  mfd_xpdr_set_auto: boolean;
+  /** (MFD SURV -> RMP) Altitude reporting ON/OFF */
+  mfd_xpdr_set_alt_reporting: boolean;
+  /** (MFD SURV -> TCAS) TCAS Alert Level 0 - STBY | 1 - TA ONLY | 2 - TARA */
+  tcas_alert_level: number;
+  /** (MFD SURV -> TCAS) TCAS Alt Select 0 - NORM | 1 - ABV | 2 - BLW */
+  tcas_alt_select: number;
```

Comment:
These should at least have `mfd_` prefix to identify the publisher and avoid conflicts.